go.mod

```diff
@@ -1,48 +1,49 @@
 module github.com/lightninglabs/taproot-assets
 
-go 1.21
+go 1.21.4
```

Comment:
I think this should remain `go 1.21`, since a minor version apparently isn't valid in some cases.

Reply:
I think lnd master requires `go 1.21.4`, which won't allow us to use `go 1.21` here